read-the-docs-2018-stats.rst

```diff
@@ -0,0 +1,110 @@
+.. post:: Jan 7, 2019
+   :tags: stats, year-in-review
+
+Read the Docs 2018 Stats
+========================
+
+2018 was another good year for Read the Docs.
+We've settled into a sustainability model that is working for us,
+and have a team of 5 folks working full-time on the project.
+
+Here are our stats for the past year,
+which we've published for the past 6 years.
+This is part of our effort to be transparent in our organization,
+as well as our source code.
+
+.. note:: 
+
+	You can always see our stats for the last `30 days`_. 
+
+	Our posts from 2013_, 2014_, 2015_, 2016_, and 2017_ are also available.
+
+.. _30 days: http://www.seethestats.com/site/readthedocs.org
+.. _2013: https://blog.readthedocs.com/read-the-docs-2013-stats/
+.. _2014: https://blog.readthedocs.com/read-the-docs-2014-stats/
+.. _2015: https://blog.readthedocs.com/read-the-docs-2015-stats/
+.. _2016: https://blog.readthedocs.com/read-the-docs-2016-stats/
+.. _2017: https://blog.readthedocs.com/read-the-docs-2017-stats/
+
+Page Views
+----------
+
+Our stats:
+
+* 400 Million Page Views (up from 338M)
+* 77 Million Unique Visitors (up from 75M)
+
+.. From Google Analytics
+
+These numbers are not as accurate in the past, since we :doc:`implemented Do Not Track support </do-not-track>` in May, our analytics pageview figures understate our actual traffic.
+
+Site Stats
+----------
+
+The stats, in total numbers:
+
+* 100,000 projects (up from 77k)
+* 155,000 users (up from 103k)
+
+We have been battling spam quite heavily again this year,
+so these numbers might be a bit skewed.
+
+.. Project.objects.count()
+.. User.objects.count()
+
+Community
+---------
+
+This year, we had:
+
+* 28 `people`_ who committed code to the main repository (up from 23)
+* 3774 `commits`_ (up from 1058)
+* 797 `issues`_ - 189 open, 608 closed (up from 513)
+
+You can see the number of commits and issues has gone up quite a bit.
+This is largely because of the additional staffing we have had on the project,
+allowing us to be more active to both our users and to release more features.
```

Comment:
I like the idea of putting these into percentage change basis as well, just to describe some of these figures in an easier way. Commits increasing 256% is pretty incredible.